src/bond_hybrid.cpp

```diff
@@ -135,6 +142,19 @@ void BondHybrid::compute(int eflag, int vflag)
     }
   }
 
+  // if bond type can be set to zero and deleted, update bondlist_orig
```

Comment:
can you make this a more descriptive (few line) comment about the need for orig_map and the issue it is solving - i.e. to propagate a broken bond from the temporary list to original list, so that bond is truly broken, else it could "reappear" when neigh lists are rebuilt

Reply:
Yes, that sounds like a good idea. Do you think the new comment is sufficiently clear?

Reply:
yes, looks good - I just approved the PR